warehouse/admin/admin.go

```diff
@@ -53,7 +56,7 @@ func (a *Admin) Register() {
 
 // TriggerUpload sets uploads to start without delay
 func (*Admin) TriggerUpload(off bool, reply *string) error {
-	startUploadAlways.Store(!off)
+	router.StartUploadAlways.Store(!off)
```

Comment:
These calls to global functions or variables are exactly what we need to get rid of :sweat_smile: 
The fact that the `TriggerUpload` method doesn't have a method receiver can be a smell.

Reply:
See if [it makes sense](https://github.com/rudderlabs/rudder-server/pull/3945/files).

Reply:
Yes.